### ISSUE TITLE
Crash

### ISSUE THREAD

**Issue by nzAily** [DavyCraft648/Customies-NG#12]
```
[AsyncWorker#0 thread/CRITICAL]: Error: "Typed property customiesdevs\customies\block\BlockPalette::$states must not be accessed before initialization" (EXCEPTION) in "F:/PocketMine-MP/plugins/Customies_dev-26.phar/src/block/BlockPalette" at line 95
--- Stack trace ---
  #0 F:/PocketMine-MP/plugins/Customies_dev-26.phar/src/task/AsyncRegisterBlocksTask(42): customiesdevs\customies\block\BlockPalette->sortStates()
  #1 pmsrc/src/scheduler/AsyncTask(81): customiesdevs\customies\task\AsyncRegisterBlocksTask->onRun()
  #2 (): pocketmine\scheduler\AsyncTask->run()
--- End of exception information ---
[12:31:56.573] [Server thread/CRITICAL]: pocketmine\thread\ThreadCrashException: "Worker 0 crashed while running task customiesdevs\customies\task\AsyncRegisterBlocksTask#177188" (EXCEPTION) in "pmsrc/src/scheduler/AsyncPool" at line 226
--- Stack trace ---
  #0 pmsrc/src/scheduler/AsyncPool(266): pocketmine\scheduler\AsyncPool->checkCrashedWorker(int 0, object customiesdevs\customies\task\AsyncRegisterBlocksTask#177188)
  #1 pmsrc/src/scheduler/AsyncPool(241): pocketmine\scheduler\AsyncPool->collectTasksFromWorker(int 0)
  #2 pmsrc/src/Server(1855): pocketmine\scheduler\AsyncPool->collectTasks()
  #3 pmsrc/src/Server(1740): pocketmine\Server->tick()
  #4 pmsrc/src/Server(1110): pocketmine\Server->tickProcessor()
  #5 pmsrc/src/PocketMine(355): pocketmine\Server->__construct(object pocketmine\thread\ThreadSafeClassLoader#6, object pocketmine\utils\MainLogger#2, string[26] F\PocketMine-MP\, string[34] F:\PocketMine-MP\plugins\)
  #6 pmsrc/src/PocketMine(378): pocketmine\server()
  #7 F:/PocketMine-MP/PocketMine-MP.phar(168): require(string[112] phar://C:/Users/Edward Maulidul/AppData/Local/Temp/PocketMine-MP-phar-cache.0/PM)
--- End of exception information ---
[12:31:56.573] [Server thread/EMERGENCY]: An unrecoverable error has occurred and the server has crashed. Creating a crash dump
[12:31:56.607] [Server thread/EMERGENCY]: Please upload the "F:/PocketMine-MP/crashdumps/Fri_Sep_20-12.31.56-UTC_2024.log" file to the Crash Archive and submit the link to the Bug Reporting page. Give as much info as you can.
[12:31:56.608] [Server thread/EMERGENCY]: Forcing server shutdown
```

**Comment by nzAily** [comment]
Command output | This server is running PocketMine-MP
Command output | Server version: 5.18.1+dev (git hash: f183ccc7c038e39b637144b4f899a16e2b0fea21-dirty)
Command output | Compatible Minecraft version: 1.21.30 (protocol version: 729)
Command output | PHP version: 8.2.17
Command output | PHP JIT: not supported
Command output | Operating system: win

**Comment by Azvyl** [comment]
isn't this the same as #11? it was fixed in [build 28](https://poggit.pmmp.io/ci/DavyCraft648/Customies-NG/Customies/dev:28)

**Comment by nzAily** [comment]
> Owner

Different, pocketmine ng update again :'

**Comment by Azvyl** [comment]
hm? I don't really understand what you mean
you can just download the latest Customies to fix it, you still haven't updated it

**Comment by nzAily** [comment]
> hm? I don't really understand what you mean you can just download the latest Customies to fix it, you still haven't updated it

I mean, so yesterday it was fixed and then mc 1.21.30 came out and pocketmine from ng was updated and then this error appeared
sorry if my English is bad

**Comment by nzAily** [comment]
fixed in the fork
